recipes/llvmdev38/build.sh

```diff
@@ -0,0 +1,38 @@
+CMAKE_COMMON_VARIABLES=" -DCMAKE_INSTALL_PREFIX=$PREFIX/lib/llvm-3.8 \
```

Comment:
Honestly maybe this should just install `$PREFIX` without messing around with it at all. If we are just build static libraries, this will only be a build time dependency. For that matter, version info isn't necessary either as there should be no use case for installing multiple copies of the same static libraries at the same time. Right @pitrou?